docs/website/content/v0.4/en/guides/getting-started/firecracker.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Firecracker
+---
+
+In this guide we will create a Kubernetes cluster using Firecracker.
+
+## Requirements
+
+- Linux
+- a kernel with KVM enabled (`/dev/kvm` must exist)
+- at least `CAP_SYS_ADMIN` and `CAP_NET_ADMIN` capabilities
+- firecracker (v0.21.0 or higher)
+- `bridge`, and `firewall` CNI plugins from the [standard CNI plugins](https://github.com/containernetworking/cni), and `tc-redirect-tap` CNI plugin from the [Firecracker Go SDK](https://github.com/firecracker-microvm/firecracker-go-sdk/tree/master/cni) installed to `/opt/cni/bin`
+- iptables
+- `/etc/cni/conf.d` directory should exist
+- `/var/run/netns` directory should exist
+
+## Create the Cluster
+
+```bash
+sudo talosctl cluster create --wait --provisioner firecracker
+```
+
+Once the above finishes successfully, your talosconfig(`~/.talos/config`) will be configured to point to the new cluster.
+
+## Retrieve and Configure the `kubeconfig`
+
+```bash
+talosctl kubeconfig .
+kubectl --kubeconfig kubeconfig config set-cluster talos-default --server https://127.0.0.1:6443
+```
+
+## Using the Cluster
+
+Once the cluster is available, you can make use of `talosctl` and `kubectl` to interact with the cluster.
```

Comment:
one more thing we might want to add is output of `talosctl cluster show --provisioner firecracker` which shows the networking side

Reply:
Firecracker doesn't work on my laptop unfortunately :/

Reply:
I'll get default output with 3 +1 for you

Reply:
```
$ talosctl cluster show --provisioner firecracker
PROVISIONER       firecracker
NAME              talos-default
NETWORK NAME      talos-default
NETWORK CIDR      10.5.0.0/24
NETWORK GATEWAY   10.5.0.1
NETWORK MTU       1500

NODES:

NAME                     TYPE           IP         CPU    RAM      DISK
talos-default-master-1   Init           10.5.0.2   1.00   1.6 GB   4.3 GB
talos-default-master-2   ControlPlane   10.5.0.3   1.00   1.6 GB   4.3 GB
talos-default-master-3   ControlPlane   10.5.0.4   1.00   1.6 GB   4.3 GB
talos-default-worker-1   Join           10.5.0.5   1.00   1.6 GB   4.3 GB
```

Reply:
Thanks!